packages/server/database/migrations/20230801200811-fixOrganizationTier.ts

```diff
@@ -0,0 +1,9 @@
+import {R} from 'rethinkdb-ts'
+
+export const up = async function (r: R) {
+  r.table('Organization').filter(r.row('tier').eq('personal')).update({tier: 'starter'}).run()
+}
+
+export const down = async function (r: R) {
+  // noop
```

Comment:
Didn't implement a `down` function, as any cleanup should be handled by `20221220161224-updateTier.ts`.